pymemcache/serde.py

```diff
@@ -1,98 +1,10 @@
-# Copyright 2012 Pinterest.com
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+"""
+Backwards compatibility with the older serialization api previously
+provided by this module.
+"""
+from . import codecs
```

Comment:
PEP8 allows relative imports. Is there a reason to not use them and opt for the less featureful alternative?